libcontainerd/remote_linux.go

```diff
@@ -254,6 +256,11 @@ func (r *remote) handleEventStream(events containerd.API_EventsClient) {
 	for {
 		e, err := events.Recv()
 		if err != nil {
+			if strings.Contains(err.Error(), "transport is closing") &&
```

Comment:
Yes, I agree, inspect error string is stupid way, this looks better.
At first I thought it is `transport.ErrConClosing.Desc` but it's not, it's `rpcError` with Description from `transport.ErrConClosing`. I was hoping to find some function like `IsErrConClosing` but there's not something like this. :sob: